zipkin-autoconfigure/metrics-prometheus/pom.xml

```diff
@@ -52,11 +52,29 @@
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
 
-        <!-- Test dependencies -->
+        <!-- Prometheus client -->
         <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-test</artifactId>
-            <scope>test</scope>
+          <groupId>io.prometheus</groupId>
+          <artifactId>simpleclient</artifactId>
+          <version>0.0.23</version>
```

Comment:
unstable I meant the patch-only version number :)

nit: maybe use a property to coordinate these? ex.

```
in the properties section
<simpleclient.version>0.0.23</simpleclient.version>

then in places like here:
${simpleclient.version}

```